build-dist.bat

```diff
@@ -0,0 +1,18 @@
+@echo off
+echo Building dist
+IF exist .\dist (rmdir /s /q .\dist)
+mkdir .\dist
+xcopy /q /y /v /i /s /e .\quantinuum_sphinx\* .\dist
+echo "Generating UI assets..."
+cd .\react
+call npm ci
+call npm run build
+echo "Copying UI assets to dist..."
+copy /y /v .\build\injectNav.global.js ..\dist\_static\injectNav.global.js
+copy /y /v .\build\injectFooter.global.js ..\dist\_static\injectFooter.global.js
```

Comment:
This addition is missing in `build-dist.sh`. Keeping the build logic to one file should prevent this drift.